src/components/command-bar/components/activator/command-bar-activator.module.css

```diff
@@ -47,6 +47,12 @@
 
 		/* TODO update the light version with a token instead of manual box shadow */
 		box-shadow: unset;
+
+		/* Ensures the g-focus-ring-from-box-shadow-dark styles aren't overridden */
+		&:focus-visible {
```

Comment:
I can't use `compose` in here, so I just copied the styles from `g-focus-ring-from-box-shadow-dark`